subprojects/gradle/signer/src/main/kotlin/com/avito/plugin/SignArtifactTask.kt

```diff
@@ -48,9 +48,9 @@ abstract class SignArtifactTask @Inject constructor(objects: ObjectFactory) : De
             timeProvider = timeProvider,
             loggerFactory = loggerFactory
         ).provide()
-            .connectTimeout(TIMEOUT_SEC, TimeUnit.SECONDS)
-            .writeTimeout(TIMEOUT_SEC, TimeUnit.SECONDS)
-            .readTimeout(TIMEOUT_SEC, TimeUnit.SECONDS)
+            .connectTimeout(10L, TimeUnit.SECONDS)
+            .writeTimeout(40L, TimeUnit.SECONDS)
+            .readTimeout(40L, TimeUnit.SECONDS)
```

Comment:
Why so big timeout for reading? Do we see the problem in metrics?

Reply:
If I'm not mistaken read timeout defines a maximum time of inactivity between two data packets when waiting for the server's response. If so, it doesn't depend on file size.

Reply:
yes, i just added a big enough value to fix it, and then gather metrics to set correct value